inline_scan.sh

```diff
@@ -24,7 +24,8 @@ DOCKERFILE="./Dockerfile"
 TIMEOUT=300
 TMP_PATH="/tmp/sysdig"
 # Analyzer option variable defaults
-SYSDIG_BASE_SCANNING_URL="https://api.sysdigcloud.com"
+SYSDIG_BASE_SCANNING_API_URL="https://api.sysdigcloud.com"
```

Comment:
Shouldn't we use the new SYSDIG_BASE_SCANNING_API_URL variable in the `curl` requests? I don't see any usage.

Reply:
Sorry, I noticed it is used in line 143 later.

Reply:
Sorry, we use SYSDIG_SCANNING_URL which is composed from this one.

Reply:
The variable SYSDIG_BASE_SCANNING_API_URL is used for setting SYSDIG_SCANNING_URL